Documentation/Development.md

```diff
@@ -66,7 +66,7 @@ using a [snapshot](https://swift.org/download/#releases) from swift.org.
 
     Note: The bootstrap script requires having [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) installed. Please refer to the [Swift project repo](https://github.com/apple/swift/blob/master/README.md#macos) for installation instructions.
 	
-    This command will build the Package Manager inside `.build/` directory.
+    This command will build the Package Manager inside the `.build/` directory.
```

Comment:
I think we can use present tense here, changing it to "This command builds the Package Manager..." while we're changing it.

Reply:
Agreed. This has been pushed into updated.